app/controllers/sessions_controller.rb

```diff
@@ -1,13 +1,21 @@
 # frozen_string_literal: true
 class SessionsController < Devise::SessionsController  
     skip_before_action :verify_authenticity_token
-    
     respond_to :json
     private
     def respond_with(resource, _opts = {})
-        render json: resource
+        resJson = {
```

Comment:
Try to use better names of variables;)
Res = Carne de vaca

Reply:
thank you bro